### DISCUSSION TITLE
Sponsored Brands' cost breakdown on per ASIN/SKU level

### DISCUSSION THREAD

**Discussion by adeelsohailahmed** [amzn/ads-advanced-tools-docs#40] [Q&A] [answered]
Hello,

I'm looking to get the overall sponsored brands cost/spend for an individual product for a given day. Basically, how much did it cost the seller to advertise a particular product via Sponsored Brands ad type for a given day.

I've been able to get the campaign, ad groups, and even individual keyword cost/spend for a day via Sponsored Brands' reports, yet I haven't been to get the cost per ASIN or SKU. 

Cost breakdown on campaign and/or ad group level is too broad for uses. On the other hand, the individual keyword cost breakdown is of no use to us if we can't reconcile those keywords with the ASINs they've been created for.

- Is there some way where I could get the Sponsored Brands' cost breakdown on per ASIN/SKU level via the API?
- If the above is not possible, is there some way where we can get the ASINs list in response for one or more given keywords?

I'm completely at a loss here. I've scoured the Amazon Advertisement API several times, but couldn't find anything. Any help will be greatly appreciated. Thanks in advance.

**Comment by elizafarley** [comment, marked as answer]
Hi @adeelsohailahmed, you should be able to get this granularity using the Sponsored Brands `ads` report type. You can view the available metrics [here](https://advertising.amazon.com/API/docs/en-us/reporting/v2/report-types#ad-reports). While this report doesn't provide a breakdown by ASIN/SKU, you can get metrics broken out at the creative/ad level. You can then use the [POST sb/beta/ads/list](https://advertising.amazon.com/API/docs/en-us/sponsored-brands/3-0/openapi/prod#/Ads/ListSponsoredBrandsAds) endpoint to see all of your ads/creatives and their associated ASINs.

Note that we have [released a new version](https://advertising.amazon.com/API/docs/en-us/sponsored-brands/campaigns/v4-migration-guide) (in beta) of the SB campaign management APIs that allows you to create multiple ad groups and ads within a single campaign. Ads created using the version 4 endpoints have an `adId`, while ads created using version 3 are identified by the `campaignId`/`adGroupId` combination. Therefore in the `ads` report, you will only see `adId` for ads created using the version 4 endpoints.

**Reply by adeelsohailahmed** [reply]
Hi @elizafarley, thank you for your quick reply. I'll give this a try and get back to you.

**Reply by adeelsohailahmed** [reply]
How do we determine which ASIN performed better and/or costed more than the other ASINs in the same creative/ad level, though? Is it currently possible via the API? If not, are there any plans to implement this in the future?

**Reply by elizafarley** [reply]
Unfortunately, there is currently no way to isolate individual ASINs used within the same creative. If this feature becomes available, we'll post in the [Release notes](https://advertising.amazon.com/API/docs/en-us/info/release-notes).

**Reply by adeelsohailahmed** [reply]
Thanks, @elizafarley! You've been a great help.
